brave/brave-core-crx-packager
# Remove ad-block (cpp) and fix packaging for new ad-block lists

Fix #143 
Fix https://github.com/brave/brave-core-crx-packager/issues/131
Fix #130 

- removes ad-block and related CPP libs (easier to keep up with node, hurray)
- removes other not currently used lists (extension list, Tracking protection list, etc)
- fixes component packaging for new ad-block lists

## Test Plan:

1. Open Brave Browser with `--use-dev-goupdater-url` 
2. Navigate to brave://components and verify `Ad-block Updater` and `Local Component Updater` download works
3. Navigate to cnn.com - and confirm that ads are blocked
4. Navigate to https://chrome.google.com/webstore/detail/1password-extension-deskt/aomjjhallfgjeglblehebfpbcfeobpgk - Install and confirm the extension warning is not displayed
5. Navigate to `<user-dir>/cffkpbalmllkdoenhmdmpbkajipdjfam` - Confirm the `cpp dat` files are not present
6. Navigate to `<user-dr>/afalakplffnnnlkncjhbmahjfjhmlkal/<version>/1` - Confirm that `TrackingProtection.dat` and `StorageTrackingProtection.dat` files are not present
7. Navigate to brave://adblock, select all - and confirm that all components download without any issues
